tensorflow_probability/python/experimental/stats/sample_stats.py

```diff
@@ -167,15 +166,31 @@ def initialize(self):
     return RunningCovarianceState(**init_state._asdict())
 
   def update(self, state, new_sample, axis=None):
-    """Update the `RunningCovarianceState` with a new sample using the update
-    formula proposed by Philippe Pebay (2008).
-
-    This implementation supports both batched and chunked covariance
-    computation. A "batch" is the usual parallel computation, namely a batch
-    of size N implies N independent covariance computations, each stepping one
-    sample (or chunk) at a time. A "chunk" of size M implies incorporating M
-    samples into a single covariance computation at once, which is more
-    efficient than one by one.
+    """Update the `RunningCovarianceState` with a new sample.
+
+    The update formula is from Philippe Pebay (2008). This implementation
+    supports both batched and chunked covariance computation. A "batch" is the
+    usual parallel computation, namely a batch of size N implies N independent
+    covariance computations, each stepping one sample (or chunk) at a time. A
+    "chunk" of size M implies incorporating M samples into a single covariance
+    computation at once, which is more efficient than one by one.
+
+    To further illustrate the difference between batching and chunking, consider
+    the following example:
+
+    ```python
+    import tensorflow as tf
+    import tensorflow_probability as tfp
+    # treat as 3 independent random variables of shape (5, 2), where each of
+    # those random variables represent 5 shape (2,) samples from the same chain
```

Comment:
Wait, wouldn't this be 3 samples from each of 5 independent vector random variables of shape (2,)?